pages/runtime/data-api.md

```diff
@@ -0,0 +1,672 @@
+# Data API Service
+
+## Introduction
+Resin's Data API Service constitutes a central node in resin architecture, since it provides interconnection between resin services and communication with the database. At the same time, it exposes an HTTP interface, based on Open Data ([OData](http://www.odata.org)) format, that provides a means to the user to interact with resin resources and fetch or update information regarding their applications, devices and environment variables.
+
+This document aims to describe the endpoints that are provided by the Data API, as well as showing examples of their expected usage.
+
+The requests are expected to follow the OData format, and the responses are provided in JSON format. The authorization token that is required in each request can be found under 'Preferences' in the user's dashboard.
+
+### Base URL
+The base URL for accessing the Data API is:
+
+`https://api.resin.io/v1/`
```

Comment:
You can define this as a variable in the documents front-matter (add this snippet to the very top of the doc):

```
---
api_base: https://api.resin.io/v1/
---
```
)

and then reuse it across the document as `{{ api_base }}`